tests/particles/test_particle_class.py

```diff
@@ -1573,3 +1573,27 @@ def test_undefined_mass_energy() -> None:
     """Test that a particle with an undefined mass energy returns |nan| J."""
     nu_tau_particle = Particle("nu_tau")
     assert u.isclose(nu_tau_particle.mass_energy, np.nan * u.J, equal_nan=True)
+
```

Comment:
Can you add a couple additional tests here: 

1) Ensure that the ionization energy for `U-235 +1` == `U-238 +1` == `U +11
2) Ensure that `C +0` and `C` return the same ionization energy

Reply:
@pheuer it is certainly possible I'm misunderstanding something (I have a more-than-basic understanding of elements, isotopes, ions, etc, but ultimately am just a SWE looking for ways to volunteer and help scientific research 😅), but I don't think that NIST has data on isotopes such as `U-235` or `U-238`. If they do, I can't figure out how to input it.

Submitting `U-235` [results](https://physics.nist.gov/cgi-bin/ASD/ie.pl?spectra=U-235&submit=Retrieve+Data&units=1&format=0&order=0&at_num_out=on&sp_name_out=on&ion_charge_out=on&el_name_out=on&seq_out=on&shells_out=on&level_out=on&ion_conf_out=on&e_out=0&unc_out=on&biblio=on) in `Element U was given with negative electric charge. It is not allowed in this query`, suggesting they're interpreting this as U with a bunch of extra electrons rather than the intended isotope. 

On the other hand, just submitting `U` gives [a list of ions](https://physics.nist.gov/cgi-bin/ASD/ie.pl?spectra=U&submit=Retrieve+Data&units=1&format=0&order=0&at_num_out=on&sp_name_out=on&ion_charge_out=on&el_name_out=on&seq_out=on&shells_out=on&level_out=on&ion_conf_out=on&e_out=0&unc_out=on&biblio=on), but doesn't appear to list any isotopes (or perhaps I'm just not recognizing how they're differentiating them as an isotope, which is definitely possible)

Reply:
Nevermind, I just realized your other comment inadvertently answered my question here, haha